### PR TITLE
Support for non-table model types (2.x)

### PR DESCRIPTION
**Pull request by asgraf**
#951

### CODE REVIEW

File: src/Command/TestCommand.php [hunk 1]
```diff
@@ -459,12 +460,15 @@ public function generateFixtureList($subject): array
     /**
      * Process a model, pull out model name + associations converted to fixture names.
      *
-     * @param \Cake\ORM\Table $subject A Model class to scan for associations and pull fixtures off of.
+     * @param @param \Cake\Datasource\RepositoryInterface $subject A Model class to scan for associations and pull fixtures off of.
```
Comment (dereuromark) [inline src/Command/TestCommand.php]: Phpstan found the double param here invalid

**Comment by markstory** [comment]
Would be good to have test coverage for generating non-ORM models if that's going to be a feature.

**Comment by dereuromark** [comment]
ping @asgraf

**Comment by dereuromark** [comment]
Ping @cakephp/core

**Pull request closed without merging**